dune-project

```
@@ -7,7 +7,7 @@
 (generate_opam_files true)
 
 (source (github Lysxia/coq-simple-io))
-(license MIT)
+(license LGPL)
```

This should be a SPDX identifier.
```suggestion
(license LGPL-2.1-only)
```
I'm still asking for instructions: https://coq.zulipchat.com/#narrow/stream/237663-coq-community-devs-.26-users/topic/Advising.20against.20licenses.3F/near/279029933